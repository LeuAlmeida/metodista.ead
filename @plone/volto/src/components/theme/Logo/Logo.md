Logo example source, with `intl` auto-injected.

```jsx static
<Logo />
```

Output:

```jsx noeditor
var IntlProvider = require('react-intl').IntlProvider;

<div className={'rsg--pre-42'}>
  <IntlProvider locale="en">
    <Logo />
  </IntlProvider>
</div>;
```
