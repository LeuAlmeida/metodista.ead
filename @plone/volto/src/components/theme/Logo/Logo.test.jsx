import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';
import { MemoryRouter } from 'react-router-dom';

import Logo from './Logo';

const mockStore = configureStore();

describe('Logo', () => {
  it('renders a logo component', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });
    const component = renderer.create(
      <Provider store={store}>
        <MemoryRouter>
          <Logo />
        </MemoryRouter>
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
