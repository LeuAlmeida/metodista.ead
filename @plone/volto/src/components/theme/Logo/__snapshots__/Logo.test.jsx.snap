// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders a logo component 1`] = `
<a
  href="/"
  onClick={[Function]}
  title="Site"
>
  <img
    alt="Plone Site"
    className="ui image"
    height={64}
    src="Logo.svg"
    title="Plone Site"
  />
</a>
`;
